mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala

```diff
@@ -967,4 +968,122 @@ object LogisticRegressionSuite {
     "standardization" -> false,
     "threshold" -> 0.6
   )
+
+  def generateLogisticInputAsList(
```

Comment:
Could you explain why we need to copy the implementation instead of wrapping it?